public/data/AnanjayGurjar.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "Ananjay Gurjar",
+    "type": "personal",
+    "bio": "Android Developer who loves to explore various tech",
+    "avatar": "http://github.com/AnanjayGurjar.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/AnanjayGurjar",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AnanjayGurjar",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/ananjay-gurjar-86ab061b2/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Began working with GDSC-RIT",
+        "date": "May 2022",
+        "icon": "github",
+        "color": "#0a0a23",
+        "description": "Began working as a core member for RIT chapter of Google Developer Student Club, a non-profit organisation that aims to enhances the coding culture in college by organizing various events",
+        "url": "https://github.com/DSC-RIT"
+      },
+      {
+        "title": "Started as Android Intern at Meduto",
+        "date": "Jan 2022",
+        "icon": "android",
+        "color": "#3a3240",
+        "description": "Worked with Meduto as Android Developer Intern where I worked on the ",
```

Comment:
This sentence seems to be incomplete, please complete it or modify it! 🛠

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient